core/comments/comment_view.ts

```diff
@@ -759,22 +758,18 @@ css.register(`
 .blocklyDeleteIcon {
   width: 20px;
   height: 20px;
-  display: none;
-  fill: var(--commentIconColour);
```

Comment:
This should work fine, I think the problem is that you're overriding this because the actual svg has "fill" as an inline attribute.

Reply:
Alright, that's annoying and I hate all of the weird tricks to make it work :(

Reply:
Yeah it sucks, I also tried using `<use>` tags instead, but those just don't show up at all.